pkg/testutil/testutil.go

```diff
@@ -21,20 +21,23 @@ func CaptureStdout(f func()) string {
 	}()
 	os.Stdout = writer
 	log.SetOutput(writer)
-	out := make(chan string)
+	out := make(chan string, 1)
 	wg := new(sync.WaitGroup)
 	wg.Add(1)
+	var ioCopyErr error
 	go func() {
 		var buf bytes.Buffer
-		wg.Done()
-		_, err = io.Copy(&buf, reader)
-		if err != nil {
-			panic(err)
-		}
+		defer func() {
```

Comment:
Very nit but we could simplify this by calling wg.Done directly without an anonymous function! It should still create a closure and defer calling it.

```suggestion
		defer wg.Done()
```